### PR TITLE
Add Channel Webhook Events

### PR DESCRIPTION
**Pull request by aglensmith**
# [DEVDOCS-2762](https://jira.bigcommerce.com/browse/DEVDOCS-2762)

## What changed?
* Added the `store/channel` webhook events
* Put event sections in alphabetical order
* Changed title from "Webhooks" to "Webhook Events"
* Removed old anchor links

### CODE REVIEW

File: docs/api-docs/getting-started/webhooks/webhook-events.md [hunk 1]
```diff
@@ -59,7 +39,289 @@
 }
 ```
 
-<a id="webhook-events_orders"></a>
+| Name | Definition |
+| -- | -- |
+| `scope` | The [event](/api-docs/getting-started/webhooks/webhook-events) registered when the webhook was created. |
+| `store_id` | A numerical identifier that is unique to each store. |
+| `data` | A lightweight description of the [event](/api-docs/getting-started/webhooks/webhook-events) that triggered the webhook. Will vary depending on the event registered. |
+| `hash` | The payload data json encoded then passed through SH1 encryption. |
```
Comment (ektravel) [inline docs/api-docs/getting-started/webhooks/webhook-events.md:47]:
```suggestion
| `hash` | The payload data JSON encoded then passed through SH1 encryption. |

```

Update per [Naming Guidelines](https://intranet.bigcommerce.com/pages/viewpage.action?spaceKey=SST&title=Naming+Guidelines).

File: docs/api-docs/getting-started/webhooks/webhook-events.md [hunk 2]
```diff
@@ -59,7 +39,289 @@
...
 ```
 
-<a id="webhook-events_orders"></a>
+| Name | Definition |
+| -- | -- |
+| `scope` | The [event](/api-docs/getting-started/webhooks/webhook-events) registered when the webhook was created. |
+| `store_id` | A numerical identifier that is unique to each store. |
+| `data` | A lightweight description of the [event](/api-docs/getting-started/webhooks/webhook-events) that triggered the webhook. Will vary depending on the event registered. |
+| `hash` | The payload data json encoded then passed through SH1 encryption. |
+| `created_at` | Unix timestamp of the date the hook was created. |
```
Comment (ektravel) [inline docs/api-docs/getting-started/webhooks/webhook-events.md:48]:
```suggestion
| `created_at` | Hook creation date as a Unix timestamp.|

```

File: docs/api-docs/getting-started/webhooks/webhook-events.md [hunk 3]
```diff
@@ -59,7 +39,289 @@
...
+| `producer` | Will always follow the pattern `stores/store_hash`. This is the store that created the webhook. |
```
Comment (ektravel) [inline docs/api-docs/getting-started/webhooks/webhook-events.md:49]:
```suggestion
| `producer` | The store that created the webhook. Will always follow the pattern `stores/store_hash`. |

```

File: docs/api-docs/getting-started/webhooks/webhook-events.md [hunk 4]
```diff
@@ -59,7 +39,289 @@
...
+    },
+    "hash": "352e4afc6dd3fc85ea26bfdf3f91852604d57528",
+    "created_at": 1561482670,
+    "producer": "stores/{store_hash}"
+}
+```
+
+| Name | Description |
+|-|-|
+| store/cart/* | Subscribe to all cart events. This will also subscribe you to cart/lineItem. |
```
Comment (ektravel) [inline docs/api-docs/getting-started/webhooks/webhook-events.md:71]:
```suggestion
| store/cart/* | Subscribe to all cart events. This will also subscribe you to `cart/lineItem`. |

```

File: docs/api-docs/getting-started/webhooks/webhook-events.md [hunk 5]
```diff
@@ -59,7 +39,289 @@
...
+| store/cart/created | This webhook will fire whenever a new cart is created, either via a storefront shopper adding their first item to the cart, or when a new cart is created via an API consumer. If it is from the storefront, then it fires when the first product is added to a new session.(The cart did not exist before). For the API it means a `POST` to /carts, (V3 and Storefront API). The `store/cart/updated` hook will also fire.|
```
Comment (ektravel) [inline docs/api-docs/getting-started/webhooks/webhook-events.md:72]:
```suggestion
| store/cart/created | This webhook will fire whenever a new cart is created, either via a storefront shopper adding their first item to the cart, or when a new cart is created via an API consumer. If it is from the storefront, then it fires when the first product is added to a new session. (The cart did not exist before.) For the API it means sending a `POST` request to `/carts`, (V3 and Storefront API). The `store/cart/updated` hook will also fire.|

```
This explanation is a bit hard to follow. Maybe simplify or shorten the sentences to make them easier to comprehend. For example:

This webhook will fire whenever a new cart is created either though a storefront or by sending a `POST` request to `/carts` (V3 and Storefront APIs). For carts created via storefront, the webhook will fire when the first product is added to a new session. (The cart did not exist before.) When you create a new cart using an API consumer, the `store/cart/updated` hook will fire in addition to `store/cart/created`.

File: docs/api-docs/getting-started/webhooks/webhook-events.md [hunk 6]
```diff
@@ -59,7 +39,289 @@
...
+| store/cart/updated | This webhook is fired whenever a cart is modified through the changes in its line items. Eg. when a new item is added to a cart or an existing item’s quantity is updated. This hook also fires when the email is changed during guest checkout or when an existing item is deleted. The payload will include the ID of the cart being updated. <br> This webhook also fires along with the cart created hook, because the first product being added to an empty cart triggers an update. It will also fire for the following events: <br> - Logging into customer account after creating a cart (email is inherited from customer account email) <br>- Entering email address via guest checkout <br> -Changing the email in guest checkout |
```
Comment (ektravel) [inline docs/api-docs/getting-started/webhooks/webhook-events.md:73]:
```suggestion
| store/cart/updated | This webhook fires whenever you modify a cart by changing its line items. For example, when you add a new item to a cart or update an existing item’s quantity. This hook also fires when you change the email during guest checkout or when you delete an existing item. The payload will include the ID of the cart being updated. <br> This webhook also fires along with the cart created hook, because the first product being added to an empty cart triggers an update. It will also fire for the following events: <br> - Logging into customer account after creating a cart (email is inherited from customer account email) <br>- Entering email address via guest checkout <br> -Changing the email in guest checkout |

```

Comment (ektravel) [reply docs/api-docs/getting-started/webhooks/webhook-events.md:73]: I tried to remove passive voice without altering the meaning.

File: docs/api-docs/getting-started/webhooks/webhook-events.md [hunk 7]
```diff
@@ -59,7 +39,289 @@
...
+| store/cart/deleted| This webhook will fire whenever a cart is deleted. This will occur either when all items have been removed from a cart and it is auto-deleted, or when the cart is explicitly removed via a `DELETE` request by an API consumer. This ends the lifecycle of the cart. The  `store/cart/updated`  webhook will also fire when the last item is removed.|
```
Comment (ektravel) [inline docs/api-docs/getting-started/webhooks/webhook-events.md:74]:
```suggestion
| store/cart/deleted| This webhook will fire whenever a user deletes a cart either by removing all items from the cart or by sending a `DELETE` request. The `store/cart/updated` webhook will also fire when the last item is removed.|

```

File: docs/api-docs/getting-started/webhooks/webhook-events.md [hunk 8]
```diff
@@ -59,7 +39,289 @@
...
+| store/cart/couponApplied | This webhook will fire whenever a new coupon code is applied to a cart. It will include the ID of the coupon code |
+| store/cart/abandoned | This webhook will fire once after a cart is abandoned. A cart is considered abandoned if no changes have been made for at least one hour after the last modified property. This hook is available for all store plans, regardless of whether the Abandoned Cart Saver feature is enabled.|
```
Comment (ektravel) [inline docs/api-docs/getting-started/webhooks/webhook-events.md:76]:
```suggestion
| store/cart/abandoned | This webhook will fire once after a user abandons a cart. A cart is considered abandoned if no changes have been made for at least one hour after the last modified property. This hook is available for all store plans, regardless of whether the Abandoned Cart Saver feature is enabled.|

```
Can you add a link to learn more about the Abandoned Cart Saver feature?

File: docs/api-docs/getting-started/webhooks/webhook-events.md [hunk 9]
```diff
@@ -59,7 +39,289 @@
...
+    "data": {
+        "type": "cart",
+        "id": "09346904-4175-44fd-be53-f7e598531b6c"
+    },
+    "hash": "352e4afc6dd3fc85ea26bfdf3f91852604d57528",
+    "created_at": 1561482670,
+    "producer": "stores/{store_hash}"
+}
+```
+### The same response is returned for the following endpoints:
```
Comment (ektravel) [inline docs/api-docs/getting-started/webhooks/webhook-events.md:113]:
```suggestion
The same response is returned for the following endpoints:

```

File: docs/api-docs/getting-started/webhooks/webhook-events.md [hunk 10]
```diff
@@ -59,7 +39,289 @@
...
+### Note
+> The `store/cart/abandoned` hook is triggered independently of the Abandoned Cart Saver feature, which is only available on [select plans](https://www.bigcommerce.com/essentials/pricing/). To learn more about the Abandoned Cart Saver, see [Using the Abandoned Cart Saver](https://support.bigcommerce.com/s/article/Using-the-Abandoned-Cart-Saver).
+
+</div>
+</div>
+</div>
+
+**Response fields**
+- type -- can be cart or cart_line_item
+- id -- ID of the cart
```
Comment (ektravel) [inline docs/api-docs/getting-started/webhooks/webhook-events.md:99]:
```suggestion
- `id` -- ID of the cart

```

File: docs/api-docs/getting-started/webhooks/webhook-events.md [hunk 11]
```diff
@@ -59,7 +39,289 @@
...
+    "created_at": 1561482670,
+    "producer": "stores/{store_hash}"
+}
+```
+### The same response is returned for the following endpoints:
+* `store/cart/couponApplied`
+
+**Response fields**
+- type -- can be cart or cart_line_item
+- id -- ID of the cart
```
Comment (ektravel) [inline docs/api-docs/getting-started/webhooks/webhook-events.md:118]:
```suggestion
- `id` -- ID of the cart

```

File: docs/api-docs/getting-started/webhooks/webhook-events.md [hunk 12]
```diff
@@ -59,7 +39,289 @@
...
+- couponId -- ID of the coupon
```
Comment (ektravel) [inline docs/api-docs/getting-started/webhooks/webhook-events.md:119]:
```suggestion
- `couponId`-- ID of the coupon

```

File: docs/api-docs/getting-started/webhooks/webhook-events.md [hunk 13]
```diff
@@ -59,7 +39,289 @@
...
+        "type": "cart",
+        "id": "09346904-4175-44fd-be53-f7e598531b6c",
+        "couponId": 1
+    },
+    "hash": "4b7297d295141b660e8db5a0d99dfcdf459fe825",
+    "created_at": 1561482761,
+    "producer": "stores/{store_hash}"
+}
+```
+### The same response is returned for the following endpoints:
```
Comment (ektravel) [inline docs/api-docs/getting-started/webhooks/webhook-events.md:135]:
```suggestion
The same response is returned for the following endpoints:

```

File: docs/api-docs/getting-started/webhooks/webhook-events.md [hunk 14]
```diff
@@ -59,7 +39,289 @@
...
+* `store/cart/converted`
+
+**Response fields**
+- type -- Will always be cart
+- id - ID of the cart
```
Comment (ektravel) [inline docs/api-docs/getting-started/webhooks/webhook-events.md:140]:
```suggestion
- `id` - ID of the cart

```

File: docs/api-docs/getting-started/webhooks/webhook-events.md [hunk 15]
```diff
@@ -59,7 +39,289 @@
...
+- orderId - ID of the order created
```
Comment (ektravel) [inline docs/api-docs/getting-started/webhooks/webhook-events.md:141]:
```suggestion
- `orderId` - ID of the order created

```

File: docs/api-docs/getting-started/webhooks/webhook-events.md [hunk 16]
```diff
@@ -59,7 +39,289 @@
...
+    "created_at": 1561486893,
+    "producer": "stores/{store_hash}"
+}
+```
+
+## Cart line item
+
+| Name | Description |
+|-|-|
+| store/cart/lineItem/* | Subscribe to all cart line item events. This webhook will fire when a change occurs to line items in the cart. This can be when items are added to a cart, removed or updated.(Ex. change to quantity, product options or price). |
```
Comment (ektravel) [inline docs/api-docs/getting-started/webhooks/webhook-events.md:161]:
```suggestion
| store/cart/lineItem/* | Subscribe to all cart line item events. This webhook will fire when a change occurs to line items in the cart. This can be when items are added to a cart, removed, or updated. For example, change to quantity, product options, or price. |

```

File: docs/api-docs/getting-started/webhooks/webhook-events.md [hunk 17]
```diff
@@ -59,7 +39,289 @@
...
+| store/cart/lineItem/created | When a new item is added to the cart  |
```
Comment (ektravel) [inline docs/api-docs/getting-started/webhooks/webhook-events.md:162]:
```suggestion
| store/cart/lineItem/created |This webhook will fire when a new item is added to the cart.  |

```

File: docs/api-docs/getting-started/webhooks/webhook-events.md [hunk 18]
```diff
@@ -59,7 +39,289 @@
...
+| store/cart/lineItem/updated | When an item’s quantity has changed or the product options change. |
```
Comment (ektravel) [inline docs/api-docs/getting-started/webhooks/webhook-events.md:163]:
```suggestion
| store/cart/lineItem/updated | This webhook will fire when an item’s quantity or product options change. |

```

File: docs/api-docs/getting-started/webhooks/webhook-events.md [hunk 19]
```diff
@@ -59,7 +39,289 @@
...
+| store/cart/lineItem/deleted | When an item is deleted from the cart|
```
Comment (ektravel) [inline docs/api-docs/getting-started/webhooks/webhook-events.md:164]:
```suggestion
| store/cart/lineItem/deleted | This weekbook will fire when an item is deleted from the cart.|

```

File: docs/api-docs/getting-started/webhooks/webhook-events.md [hunk 20]
```diff
@@ -59,7 +39,289 @@
...
+
+### The same response is returned for the following endpoints:
```
Comment (ektravel) [inline docs/api-docs/getting-started/webhooks/webhook-events.md:166]:
```suggestion
The same response is returned for the following endpoints:

```

File: docs/api-docs/getting-started/webhooks/webhook-events.md [hunk 21]
```diff
@@ -59,7 +39,289 @@
...
+* `store/cart/lineItem/created`
+* `store/cart/lineItem/updated`
+* `store/cart/lineItem/deleted`
+
+**Response fields**
+- type -- can be cart or cart_line_item
```
Comment (ektravel) [inline docs/api-docs/getting-started/webhooks/webhook-events.md:172]:
```suggestion
- `type` -- can be `cart` or `cart_line_item`

```

File: docs/api-docs/getting-started/webhooks/webhook-events.md [hunk 22]
```diff
@@ -59,7 +39,289 @@
...
+- id -- ID of the line item
```
Comment (ektravel) [inline docs/api-docs/getting-started/webhooks/webhook-events.md:173]:
```suggestion
- `id` -- ID of the line item

```

File: docs/api-docs/getting-started/webhooks/webhook-events.md [hunk 23]
```diff
@@ -59,7 +39,289 @@
...
+- cartId -- ID of the cart
```
Comment (ektravel) [inline docs/api-docs/getting-started/webhooks/webhook-events.md:174]:
```suggestion
- `cartId` -- ID of the cart

```

File: docs/api-docs/getting-started/webhooks/webhook-events.md [hunk 24]
```diff
@@ -59,7 +39,289 @@
...
+```
+
+
+
+## Category
+
+| Name | Description |
+|-|-|
+| store/category/* | Subscribe to all store/category events |
+| store/category/created | Category is created |
```
Comment (ektravel) [inline docs/api-docs/getting-started/webhooks/webhook-events.md:198]:
```suggestion
| store/category/created | Fires when a category is created. |

```

File: docs/api-docs/getting-started/webhooks/webhook-events.md [hunk 25]
```diff
@@ -59,7 +39,289 @@
...
+| store/category/updated | Category is updated |
```
Comment (ektravel) [inline docs/api-docs/getting-started/webhooks/webhook-events.md:199]:
```suggestion
| store/category/updated | Fires when a category is updated. |

```

File: docs/api-docs/getting-started/webhooks/webhook-events.md [hunk 26]
```diff
@@ -59,7 +39,289 @@
...
+| store/category/deleted | Category is deleted |
```
Comment (ektravel) [inline docs/api-docs/getting-started/webhooks/webhook-events.md:200]:
```suggestion
| store/category/deleted | Fires when a category is deleted. |

```

File: docs/api-docs/getting-started/webhooks/webhook-events.md [hunk 27]
```diff
@@ -59,7 +39,289 @@
...
+
+### The same response is returned for the following endpoints:
```
Comment (ektravel) [inline docs/api-docs/getting-started/webhooks/webhook-events.md:202]:
```suggestion
The same response is returned for the following endpoints:

```

File: docs/api-docs/getting-started/webhooks/webhook-events.md [hunk 28]
```diff
@@ -59,7 +39,289 @@
...
+
+* `store/category/created`
+* `store/category/updated`
+* `store/category/deleted`
+
+**Response fields**
+
+- type -- Will always be category
+- id -- ID of the category
```
Comment (ektravel) [inline docs/api-docs/getting-started/webhooks/webhook-events.md:211]:
```suggestion
- `id` -- ID of the category

```

File: docs/api-docs/getting-started/webhooks/webhook-events.md [hunk 29]
```diff
@@ -59,7 +39,289 @@
...
+    "hash": "dc3a47c15425d2c895dba674f86fe71a8f3b6459",
+    "created_at": 1561480214,
+    "producer": "stores/{store_hash}"
+}
+```
+## Channel
+
+| Name                    | Description                                              |
+|-------------------------|----------------------------------------------------------|
+| `store/channel/*`       | Subscribe to all `store/channel` events                    |
```
Comment (ektravel) [inline docs/api-docs/getting-started/webhooks/webhook-events.md:230]:
```suggestion
| `store/channel/*`       | Subscribe to all `store/channel` events.                    |

```

File: docs/api-docs/getting-started/webhooks/webhook-events.md [hunk 30]
```diff
@@ -59,7 +39,289 @@
...
+| `store/channel/created` | Fires when a channel is created via control panel or API |
```
Comment (ektravel) [inline docs/api-docs/getting-started/webhooks/webhook-events.md:231]:
```suggestion
| `store/channel/created` | Fires when a channel is created via control panel or API. |

```

File: docs/api-docs/getting-started/webhooks/webhook-events.md [hunk 31]
```diff
@@ -59,7 +39,289 @@
...
+| `store/channel/updated` | Fires when a channel is updated via control panel or API |
```
Comment (ektravel) [inline docs/api-docs/getting-started/webhooks/webhook-events.md:232]:
```suggestion
| `store/channel/updated` | Fires when a channel is updated via control panel or API. |

```

File: docs/api-docs/getting-started/webhooks/webhook-events.md [hunk 32]
```diff
@@ -59,7 +39,289 @@
...
+    "scope":"store/channel/created",
+    "data":{
+            "type":"channel",
+            "id":173331
+        },
+    "hash":"3f9ea420af83450d7ef9f78b08c8af25b2213637"
+ }
+ ```
+
+| Property | Description              |
```
Comment (ektravel) [inline docs/api-docs/getting-started/webhooks/webhook-events.md:247]: Why was this done as a table when the rest of the document uses list?

File: docs/api-docs/getting-started/webhooks/webhook-events.md [hunk 33]
```diff
@@ -59,7 +39,289 @@
...
+|----------|--------------------------|
+| `type`   | Will always be `channel` |
+| `id`     | The ID of the channel    |
+
+Updates to the following fields trigger a `store/channel/updated` event.
```
Comment (ektravel) [inline docs/api-docs/getting-started/webhooks/webhook-events.md:252]:
```suggestion
Updates to the following fields trigger a `store/channel/updated` event:

```

File: docs/api-docs/getting-started/webhooks/webhook-events.md [hunk 34]
```diff
@@ -59,7 +39,289 @@
...
+* `is_visible`
+* `is_enabled` (to be deprecated)
+* `config_meta`
+
+## Customer
+
+| Name | Description |
+|-|-|
+| store/customer/* | Subscribe to all store/customer events |
+| store/customer/created | A new customer is created|
```
Comment (ektravel) [inline docs/api-docs/getting-started/webhooks/webhook-events.md:267]:
```suggestion
| store/customer/created | Fires when a new customer is created. |

```

File: docs/api-docs/getting-started/webhooks/webhook-events.md [hunk 35]
```diff
@@ -59,7 +39,289 @@
...
+| store/customer/updated | Customer is updated. Does not currently track changes to the customer address. |
```
Comment (ektravel) [inline docs/api-docs/getting-started/webhooks/webhook-events.md:268]:
```suggestion
| store/customer/updated | Fires when a customer is updated. Does not currently track changes to the customer address. |

```

File: docs/api-docs/getting-started/webhooks/webhook-events.md [hunk 36]
```diff
@@ -59,7 +39,289 @@
...
+| store/customer/deleted | Customer is deleted |
```
Comment (ektravel) [inline docs/api-docs/getting-started/webhooks/webhook-events.md:269]:
```suggestion
| store/customer/deleted | Fires when a customer is deleted. |

```

File: docs/api-docs/getting-started/webhooks/webhook-events.md [hunk 37]
```diff
@@ -59,7 +39,289 @@
...
+| store/customer/address/created | Customer address is created |
```
Comment (ektravel) [inline docs/api-docs/getting-started/webhooks/webhook-events.md:270]:
```suggestion
| store/customer/address/created | Fires when a customer address is created. |

```

File: docs/api-docs/getting-started/webhooks/webhook-events.md [hunk 38]
```diff
@@ -59,7 +39,289 @@
...
+| store/customer/address/updated | Customer address is updated |
```
Comment (ektravel) [inline docs/api-docs/getting-started/webhooks/webhook-events.md:271]:
```suggestion
| store/customer/address/updated | Fires when a customer address is updated. |

```

File: docs/api-docs/getting-started/webhooks/webhook-events.md [hunk 39]
```diff
@@ -59,7 +39,289 @@
...
+
+### The same response is returned for the following endpoints:
+* `store/customer/created`
+* `store/customer/updated`
+* `store/customer/deleted`
+* `store/customer/payment/instrument/default/updated`
+
+**Response fields**
+
+- type -- Will always be customer
```
Comment (ektravel) [inline docs/api-docs/getting-started/webhooks/webhook-events.md:283]:
```suggestion
- `type` -- will always be customer

```

File: docs/api-docs/getting-started/webhooks/webhook-events.md [hunk 40]
```diff
@@ -59,7 +39,289 @@
...
+    "data": {
+        "type": "customer",
+        "id": 32
+    },
+    "hash": "8768ab15aa86c6d73c7e4c3efbaee072110ad1d2",
+    "created_at": 1561481571,
+    "producer": "stores/{store_hash}"
+}
+```
+### The same response is returned for the following endpoints:
```
Comment (ektravel) [inline docs/api-docs/getting-started/webhooks/webhook-events.md:298]:
```suggestion
The same response is returned for the following endpoints:

```

File: docs/api-docs/getting-started/webhooks/webhook-events.md [hunk 41]
```diff
@@ -59,7 +39,289 @@
...
+* `store/customer/address/created`
+* `store/customer/address/updated`
+* `store/customer/address/deleted`
+
+**Response fields**
+- type -- Will always be customer
```
Comment (ektravel) [inline docs/api-docs/getting-started/webhooks/webhook-events.md:304]:
```suggestion
- `type` -- will always be customer

```

File: docs/api-docs/getting-started/webhooks/webhook-events.md [hunk 42]
```diff
@@ -59,7 +39,289 @@
...
+- id -- ID of the customer address
```
Comment (ektravel) [inline docs/api-docs/getting-started/webhooks/webhook-events.md:305]:
```suggestion
- `id`-- ID of the customer address

```

File: docs/api-docs/getting-started/webhooks/webhook-events.md [hunk 43]
```diff
@@ -59,7 +39,289 @@
...
+- address
```
Comment (ektravel) [inline docs/api-docs/getting-started/webhooks/webhook-events.md:306]:
```suggestion
- `address`

```

File: docs/api-docs/getting-started/webhooks/webhook-events.md [hunk 44]
```diff
@@ -273,44 +534,42 @@ Changes to any of the following fields will trigger a `store/product/updated` ev
...
 
-| Name | Description |
-|-|-|
-| store/category/* | Subscribe to all store/category events |
-| store/category/created | Category is created |
-| store/category/updated | Category is updated |
-| store/category/deleted | Category is deleted |
+| Name | Description |
+|-|-|
+| store/shipment/* | Subscribe to all store/shipment events |
```
Comment (ektravel) [inline docs/api-docs/getting-started/webhooks/webhook-events.md:542]:
```suggestion
| store/shipment/* | Subscribe to all store/shipment events. |

```

File: docs/api-docs/getting-started/webhooks/webhook-events.md [hunk 45]
```diff
@@ -273,44 +534,42 @@ Changes to any of the following fields will trigger a `store/product/updated` ev
...
+| store/shipment/created | Shipment is created |
```
Comment (ektravel) [inline docs/api-docs/getting-started/webhooks/webhook-events.md:543]:
```suggestion
| store/shipment/created | Fires when a shipment is created. |

```

File: docs/api-docs/getting-started/webhooks/webhook-events.md [hunk 46]
```diff
@@ -273,44 +534,42 @@ Changes to any of the following fields will trigger a `store/product/updated` ev
...
+| store/shipment/updated | Shipment is updated |
```
Comment (ektravel) [inline docs/api-docs/getting-started/webhooks/webhook-events.md:544]:
```suggestion
| store/shipment/updated | Fires when a shipment is updated. |

```

File: docs/api-docs/getting-started/webhooks/webhook-events.md [hunk 47]
```diff
@@ -273,44 +534,42 @@ Changes to any of the following fields will trigger a `store/product/updated` ev
...
+| store/shipment/deleted | Shipment is deleted |
```
Comment (ektravel) [inline docs/api-docs/getting-started/webhooks/webhook-events.md:545]:
```suggestion
| store/shipment/deleted | Fires when a shipment is deleted. |

```

File: docs/api-docs/getting-started/webhooks/webhook-events.md [hunk 48]
```diff
@@ -273,44 +534,42 @@ Changes to any of the following fields will trigger a `store/product/updated` ev
...
 
 ### The same response is returned for the following endpoints:
```
Comment (ektravel) [inline docs/api-docs/getting-started/webhooks/webhook-events.md:547]:
```suggestion
The same response is returned for the following endpoints:

```

File: docs/api-docs/getting-started/webhooks/webhook-events.md [hunk 49]
```diff
@@ -273,44 +534,42 @@ Changes to any of the following fields will trigger a `store/product/updated` ev
...
-* `store/category/deleted`
+* `store/shipment/created`
+* `store/shipment/updated`
+* `store/shipment/deleted`
 
 **Response fields**
-
-- type -- Will always be category
-- id -- ID of the category
+- type -- Will always be shipment
```
Comment (ektravel) [inline docs/api-docs/getting-started/webhooks/webhook-events.md:554]:
```suggestion
- `type`-- will always be shipment

```

File: docs/api-docs/getting-started/webhooks/webhook-events.md [hunk 50]
```diff
@@ -273,44 +534,42 @@ Changes to any of the following fields will trigger a `store/product/updated` ev
...
+- id -- ID of the shipment
```
Comment (ektravel) [inline docs/api-docs/getting-started/webhooks/webhook-events.md:555]:
```suggestion
- `id`-- ID of the shipment

```

File: docs/api-docs/getting-started/webhooks/webhook-events.md [hunk 51]
```diff
@@ -273,44 +534,42 @@ Changes to any of the following fields will trigger a `store/product/updated` ev
...
+- orderId -- ID of the order
```
Comment (ektravel) [inline docs/api-docs/getting-started/webhooks/webhook-events.md:556]:
```suggestion
- `orderId` -- ID of the order

```

File: docs/api-docs/getting-started/webhooks/webhook-events.md (hunk 4 above)
Comment (ektravel) [inline docs/api-docs/getting-started/webhooks/webhook-events.md:71]:
```suggestion
| store/cart/* | Subscribe to all cart events. This will also subscribe you to `cart/lineItem`. |

```

File: docs/api-docs/getting-started/webhooks/webhook-events.md (hunk 5 above)
Comment (ektravel) [inline docs/api-docs/getting-started/webhooks/webhook-events.md:72]:
```suggestion
| store/cart/created | Fires when a new cart is created either via a storefront shopper adding their first item to the cart or via an API consumer. If it is from the storefront, then it fires when the first product is added to a new session.(The cart did not exist before). For the API it means a `POST` to /carts, (V3 and Storefront API). The `store/cart/updated` hook will also fire.|

```

**Review by ektravel — COMMENTED** [review]
Added my suggestions. If possible, try to remove passive voice.

**Comment by aglensmith** [comment]
@ektravel -- the stuff you're pointing out was already in the file. I just copied and pasted stuff around to make it alphabetical. Do you think we could just review my additions (channel webhooks) and address the other issues in another PR/ticket?

**Review by uwikaiddi — APPROVED** [review]
